src/compiler/diagnosticMessages.json

```diff
@@ -4954,5 +4954,25 @@
     "No value exists in scope for the shorthand property '{0}'. Either declare one or provide an initializer." :{
         "category": "Error",
         "code": 18004
+    },
+    "Add a return statement": {
+        "category": "Message",
+        "code": 95077
+    },
+    "Remove block body braces": {
+        "category": "Message",
+        "code": 95078
+    },
+    "Replace braces with parentheses": {
+        "category": "Message",
+        "code": 95079
+    },
+    "Wrap this block with parentheses": {
+        "category": "Message",
+        "code": 95080
+    },
+    "Surmise all return value": {

```

Comment:
`"Correct all return expressions"` would be best here

"surmise" is not in basic English vocabulary and we need to avoid it entirely for clarity's sake.

Reply:
From context, I think it's being used in places where we'd normally say "infer".